feeds/wifi-ax/hostapd/patches/f00-011-hapd-ubus-channel-switch.patch

```diff
@@ -0,0 +1,176 @@
+--- a/src/ap/dfs.c
++++ b/src/ap/dfs.c
+@@ -19,6 +19,7 @@
+ #include "dfs.h"
+ #include "beacon.h"
+ #include "eloop.h"
++#include "ubus.h"
+ 
+ 
+ static int dfs_get_used_n_chans(struct hostapd_iface *iface, int *seg1)
+@@ -1207,6 +1208,11 @@ static int hostapd_dfs_start_channel_swi
+ 			break;
+ 	}
+ 
++	if (!err) {
++		wpa_printf(MSG_DEBUG, "Reporting DFS event to ubus");
++		hostapd_ubus_handle_channel_switch_event(iface, HOSTAPD_UBUS_DFS_SWITCH,  channel->freq);
```

Comment:
Tested?

Reply:
I have removed this patch for now. SM crash is making it difficult to test this particular patch. I do not want to hold the client events patch because of it. This will be taken care in WIFI-1738